examples/oemof_0.2/jupyter_tutorials/3_Draw_network_graph.ipynb

```diff
@@ -0,0 +1,226 @@
+    "    \"\"\"\n",
+    "    Draw a graph. This function will be removed in future versions.\n",
+    "\n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    grph : networkxGraph\n",
+    "        A graph to draw.\n",
+    "    edge_labels : boolean\n",
+    "        Use nominal values of flow as edge label\n",
+    "    node_color : dict or string\n",
+    "        Hex color code oder matplotlib color for each node. If string, all\n",
+    "        colors are the same.\n",
+    "\n",
+    "    edge_color : string\n",
+    "        Hex color code oder matplotlib color for edge color.\n",
+    "\n",
+    "    plot : boolean\n",
+    "        Show matplotlib plot.\n",
+    "\n",
+    "    node_size : integer\n",
+    "        Size of nodes.\n",
+    "\n",
+    "    with_labels : boolean\n",
+    "        Draw node labels.\n",
+    "\n",
+    "    arrows : boolean\n",
+    "        Draw arrows on directed edges. Works only if an optimization_model has\n",
+    "        been passed.\n",
+    "    layout : string\n",
+    "        networkx graph layout, one of: neato, dot, twopi, circo, fdp, sfdp.\n",
+    "    \"\"\"\n",
+    "    if type(node_color) is dict:\n",
+    "        node_color = [node_color.get(g, '#AFAFAF') for g in grph.nodes()]\n",
+    "\n",
+    "    # set drawing options\n",
+    "    options = {\n",
+    "     'prog': 'dot',\n",
+    "     'with_labels': with_labels,\n",
+    "     'node_color': node_color,\n",
+    "     'edge_color': edge_color,\n",
+    "     'node_size': node_size,\n",
+    "     'arrows': arrows\n",
+    "    }\n",
+    "\n",
+    "    # draw graph\n",
+    "    pos = nx.drawing.nx_agraph.graphviz_layout(grph, prog=layout)\n",
+    "\n",
+    "    nx.draw(grph, pos=pos, **options)\n",
+    "\n",
+    "    # add edge labels for all edges\n",
+    "    if edge_labels is True and plt:\n",
+    "        labels = nx.get_edge_attributes(grph, 'weight')\n",
+    "        nx.draw_networkx_edge_labels(grph, pos=pos, edge_labels=labels)\n",
+    "\n",
+    "    # show output\n",
+    "    if plot is True:\n",
+    "        plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read back pickled graph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "energysystem_graph = nx.readwrite.read_gpickle('energysystem_graph')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Draw graph\n",
+    "\n",
+    "To define colors, use color palette generators, e.g.:   \n",
+    "http://javier.xyz/cohesive-colors/  \n",
+    "https://colourco.de/  \n",
+    "http://seaborn.pydata.org/tutorial/color_palettes.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Graph: edge_labels hinzufügen und Lösung mitschicken.\n",
+    "# Farbzuweisung verbessern\n",
```

Comment:
```suggestion
    "# change and improve colormap\n",
```

Reply:
Thanks for pointing to those, that were just left overs. Cleaned up now.